docs/_documentations/project-actions.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+title: Project actions
+description: Project actions
+keywords: run, open, import, show, restart, edit, build, logs, tools, project actions, attach, build, disable, enable, validate, refresh
+duration: 1 minute
+permalink: project-actions
+type: document
+order: 30
+parent: 
+---
+
+# Project actions
+
+The following actions are available by right clicking on the project in the **Codewind Explorer** view. Most actions are only available if the project is enabled.
+
+Some actions open the default browser. If you find that the default browser cannot handle the content, change the default browser by navigating to **Window** > **Web Browser** and selecting a different browser from the list.
+
+- **Open Application**: Opens the application in the default browser. This action is only available when the application is running or debugging.
+- **Project Overview**: Opens the overview page for this project. You can use this to see information about the project, enable or disable the project, turn auto build on and off, or edit project settings.
+- **Open Container Shell**: Opens a shell into your application container. This action is only available when the container is active.
+- **Application Monitor**: Opens the application monitor in the default browser. Use this to monitor the activity and health of your application. This action is only available when the application is running or debugging.
+- **Performance Dashboard**: Opens the performance dashboard in the default browser. This action is available only when the application is running or debugging.
+- **Tekton Dashboard**: This option is available to you if you are using remote connections, and only if you installed pipelines. Use this option to open the dashboard.
+- **Import Project**: Imports your project into the current workspace.
+- **Disable/Enable Project**: Disables or enables the project.
+- **Show Log Files**: If log files are available, this action displays a list of log files. Click a log file to open it, or click **Show All** to open all available log files. The individual log files are toggle actions. Click the log file again to remove the log file, or click **Hide All** to remove all log files from the **Console** view.
+- **Restart in Run Mode**: Restarts the application in run mode.
```

Comment:
This action is only available on projects that support restart in run mode. Since there are extensions and this can change it is better not to try and list them.